CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ Changelog
 ### Enhancements
 
 * Adds support for tracking sessions and crash rate by setting the configuration option `configuration.track_sessions` to `true`.
-  Sessions can be manually created using `Bugsnag.start_session`, and manually delivered using `Bugsnag.send_sessions`.
+  Sessions can be manually created using `Bugsnag.create_session`, and manually delivered using `Bugsnag.send_sessions`.
```

Comment:
Don't mention the `send_sessions` thing. Ideally that shouldn't even be public imo. It doesn't look like it's defined on Bugsnag anyway. The function should be `start_session` not `create_session`

Reply:
Ah, that's the old changelog.  I'll fix the copy.